### PR TITLE
[plugin.video.dailymotion_com] v2.3.7

### PR DESCRIPTION
**Pull request by Gujal00**
### Description
 - Fix for playing Live Video
 - Add support for limiting quality of Live Streams
 - Supoort New Qualities such as 4K and QuadHD
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Gujal00** [comment]
@Rechi 
I understand one of the errors that Language files are in old format
But there are 8 errors saying "ERROR: ./plugin.video.dailymotion_com/addon.xml is marked as stand-alone executable" which I can't control
Please advise

**Comment by Gujal00** [comment]
Thanks for the help @Rechi the travis tests have now passed successfully.

File: plugin.video.dailymotion_com/default.py [hunk 1]
```diff
@@ -265,46 +258,34 @@ def searchUser():
...
+        url=getStreamUrl(vid,live=True)
     else:
-        url = getStreamUrl(id)
+        url = getStreamUrl(vid)
     #xbmc.log("DAILYMOTION - url = %s" %url,xbmc.LOGNOTICE)
     if url:
         listitem = xbmcgui.ListItem(path=url)
         xbmcplugin.setResolvedUrl(pluginhandle, True, listitem)
     else:
         print 'No playable url found'
```
Comment (Rechi) [inline plugin.video.dailymotion_com/default.py]: Please update the print statements to the print function.
```suggestion
         print('No playable url found')
```

File: plugin.video.dailymotion_com/default.py [hunk 2]
```diff
@@ -265,46 +258,34 @@ def searchUser():
...
     else:
         return int(elem[0].split("@")[0])
 
-def getStreamUrl(id,live=False):
+def getStreamUrl(vid,live=False):
     if familyFilter == "1":
         ff = "on"
     else:
         ff = "off"
     print 'The url is ::',url
```
Comment (Rechi) [inline plugin.video.dailymotion_com/default.py]:
```suggestion
     print('The url is ::%s' % url)
```

File: plugin.video.dailymotion_com/default.py [hunk 3]
```diff
@@ -334,34 +315,52 @@ def getStreamUrl(id,live=False):
...
                         m_url = m_url.replace('dvr=true&','')
-                        if '.m3u8?auth' in m_url:
-                            m_url = m_url.split('?auth=')
-                            the_url = m_url[0] + '?redirect=0&auth=' + urllib.quote(m_url[1])
+                        if '.m3u8?sec' in m_url:
+                            m_url = m_url.split('?sec=')
+                            the_url = m_url[0] + '?redirect=0&sec=' + urllib.quote(m_url[1])
                             rr = requests.get(the_url,cookies=r.cookies.get_dict() ,headers=headers)
                             if rr.headers.get('set-cookie'):
                                 print 'adding cookie to url'
```
Comment (Rechi) [inline plugin.video.dailymotion_com/default.py]:
```suggestion
                                 print('adding cookie to url')
```

File: plugin.video.dailymotion_com/default.py [hunk 4]
```diff
@@ -334,34 +315,52 @@ def getStreamUrl(id,live=False):
...
                     
         if len(other_playable_url) >0: # probably not needed, only for last resort
             for m_url in other_playable_url:
                 #xbmc.log("DAILYMOTION - other m_url = %s" %m_url,xbmc.LOGNOTICE)
                 m_url = m_url.replace('dvr=true&','')
-                if '.m3u8?auth' in m_url:
+                if '.m3u8?sec' in m_url:
                     rr = requests.get(m_url,cookies=r.cookies.get_dict() ,headers=headers)
                     if rr.headers.get('set-cookie'):
                         print 'adding cookie to url'
```
Comment (Rechi) [inline plugin.video.dailymotion_com/default.py]:
```suggestion
                         print('adding cookie to url')
```

File: plugin.video.dailymotion_com/default.py [hunk 5]
```diff
@@ -487,7 +484,8 @@ def getUrl2(url):
         ff = "off"
     print 'The url is ::',url
```
Comment (Rechi) [inline plugin.video.dailymotion_com/default.py]:
```suggestion
     print('The url is ::%s' % url)
```

File: plugin.video.dailymotion_com/default.py (hunk 1 above)
Comment (Rechi) [reply plugin.video.dailymotion_com/default.py]: or even better replace them with `xbmc.log(...)`

File: plugin.video.dailymotion_com/default.py (hunk 5 above)
Comment (Gujal00) [reply plugin.video.dailymotion_com/default.py]: I will replace all the print statements with xbmc.log and resubmit
Those print statements are from eden days!

**Pull request merged**